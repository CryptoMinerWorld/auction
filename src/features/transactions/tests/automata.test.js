import { testStateMachine } from 'react-automata';
import Tx from '../index';

test('transactions module works', () => {
  testStateMachine(Tx);
});
