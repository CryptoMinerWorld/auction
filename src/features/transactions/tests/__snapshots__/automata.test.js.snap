// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`transactions module works: signedOut 1`] = `<div />`;

exports[`transactions module works: signedin.idle 1`] = `
<div>
  logged In
</div>
`;

exports[`transactions module works: signedin.idle 2`] = `
<div>
  <button
    onClick={[Function]}
    type="button"
  >
    x
  </button>
  logged In
</div>
`;
